core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -574,6 +611,7 @@ private IndexResult innerIndex(Index index) throws IOException {
         final Translog.Location location;
         final long updatedVersion;
         IndexResult indexResult = null;
```

Comment:
nit - this can be put in scope and next to where it's created and probably made final